docgen/content-sources/v1/toc.yaml

```diff
@@ -1,148 +1,148 @@
 toc:
   - title: 'functions'
-    path: /docs/reference/functions/v1_cloud_functions_.html
```

Comment:
This is a surprise to me.  The currently live reference docs don't use this versioned path (they're just https://firebase.google.com/docs/reference/functions/cloud_functions_#cloudfunction, etc).

Also, in the past I believe we manually created this file and then generated the usable, publishable _toc.yaml file from it.  Did you devise a way to autogenerate content-sources/toc.yaml?

Reply:
No, I manually corrected this based on the output of the script in this branch.